src/App.js

```diff
@@ -0,0 +1,200 @@
+import React, { Component } from 'react'
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import styled from 'styled-components';
+import Menu from './UI/Menu/Menu.jsx';
+import Game from './UI/Game.jsx';
+import Leaderboard from './UI/Leaderboard.jsx';
+import './main.css';
+import backgroundLight from './assets/images/bg-light.png';
+import backgroundDark from './assets/images/bg-night.png';
+import mainTheme from './assets/audio/main theme chill.mp3';
+import Callback from './UI/Callback';
+import { connect } from 'react-redux';
+import { changeTheme, changeMusic, fetchUser, fetchLeaderboard } from './store/actions';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: null,
+      leaderboard: {
+        easy: [],
+        medium: [],
+        hard: [],
+      },
+    };
+
+    this.getAllData();
+
+    this.musicVolumeHandler = this.musicVolumeHandler.bind(this);
+    this.soundVolumeHandler = this.soundVolumeHandler.bind(this);
+    this.mainTheme = new Audio(mainTheme);
+    this.mainTheme.loop = true;
+  }
+
+  getAllData() {
+    const token = localStorage.getItem('token') || null;
```

Comment:
You don't need to use `||` 'cause it will already have null value if there is no `token` in localStorage